lib/tasks/update_selectable_schools.rake

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module DataMigrations
+  class SetSelectableSchoolsOnProviders
+    def change
+      provider_codes = %w[E65 2CG 2CH 2CJ 2CK 2C1 1EX 1QU CS01 2X4 1ZO]
+      Provider.where(provider_code: provider_codes).update_all(selectable_school: true) # rubocop:disable Rails/SkipsModelValidations
```

Comment:
Do we need to update it for all 6 recruitment cycles? Could we just do it for the current and the next recruitment cycle? 

Also this updates 11 providers. in the PR description it says 7, did we find more that need to be updated?

Reply:
Also, If we need to disable the cop could we do it in:

https://github.com/DFE-Digital/publish-teacher-training/blob/main/config/rubocop/rails.yml

So we can see which cops have been excluded. 

Would using a `find_each` with `update_columns` allow you to do it without disabling any cops? Not too fussed, just wondering what your thoughts were.

Reply:
Yeah, I had 7 in my head since earlier in the process, it's actually 11.

As for the rubocop thing, I'm not too bothered about it at all. I'm happy to disable it throughout the application.

Reply:
You don't have to disable it throughout the application, I think you can just disable it for the file like this: 

https://github.com/DFE-Digital/publish-teacher-training/blob/47b710e467c8014fbef01ac2a870cbfbf04a8cf1/config/rubocop/rails.yml#L13-L15

It's just nice to see everything that's disabled in one place.

Reply:
I don't think it's a linting rule worth having. It's very well known what `update_all` does and anyone using it where they shouldn't ought to be picked up in a code review. I don't think we should have to selectively disable this rule.

Reply:
Though I agree having all the bypasses in one place is a good idea